canonical-web-and-design/ubuntu.com
# Remove security db and related files

## Done

- Refactored rss feed and added redirect for atom feed type 
- Refactored sitemaps views for cves and notices
- Removed unused imports related to the db
- Removed functionality for creating, updating, and deleting notices and cves

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/cves/sitemap.xml,  http://0.0.0.0:8001/security/notices/sitemap.xml and see that both match the live site

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/11842
